addons/common/fnc_addBinocularMagazine.sqf

```diff
@@ -0,0 +1,39 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_addBinocularMagazine
+
+Description:
+    Adds a magazine to the units rangefinder.
+    Note that this breaks the unique magazine ids due to the usage of setUnitLoadout.
```

Comment:
Add an empty line after the first sentence. The first sentence can(*) then be used to generate a `description` field for CfgFunctions entries.

(*) The old Ruby script that does this needs updating, but with the Description: paragraph in this form, automation is possible.